app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,93 @@
+<div class="container">
+  <div class="row d-flex justify-content-center">
+    <div class="col-md-8 mt-4 p-2">
+      <h1 class="text-center">
+        <%= @recipe.name %>
+      </h1>
+  
+      <div class="d-flex justify-content-between align-items-center mb-3">
+        <span>
+          Preparation Time:
+          <%= @recipe.preparation_time %>
+          <%= @recipe.preparation_time > 1 ? 'hours' : 'hour' %> 
+        </span>
+        <div data-controller="recipe"
+            data-recipe-update-url="<%= recipe_path(@recipe) %>"
+            class="form-check form-switch form-check-reverse"
+        > 
+          <input type="checkbox" 
+              data-action="recipe#toggle" data-target = "public"
+              class="form-check-input" id="flexSwitchCheckReverse"
+                <% if @recipe.public %> checked <% end %>>
+          <label class="form-check-label" for="flexSwitchCheckReverse">Public</label>
```

Comment:
- [ ] Please make sure that clicking on a recipe switch icon should switch a recipe from public to private when it is public or private to public when it is private. :pray: